ddtrace/contrib/requests/connection.py

```diff
@@ -70,6 +70,7 @@ def _wrap_send(func, instance, args, kwargs):
 
     url = request.url
     hostname = _extract_hostname(url)
+    host_without_port = hostname.split(":")[0] if hostname is not None else None
```

Comment:
are we guaranteed a port?

Reply:
We're not, but we're guaranteed that if there is one, then `urlparse` puts it in `netloc` (at least, it does if it's an absolute URL, which we make sure of).